core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -78,7 +82,48 @@ oppia.directive('explorationSummaryTile', [function() {
           }
         );
 
-        $scope.MAX_CONTRIBUTORS_TO_DISPLAY = 5;
+        $scope.avatarsList = [];
+        $scope.contributors.forEach(function(contributorName) {
+          var DEFAULT_PROFILE_IMAGE_PATH = '/images/avatar/user_blue_72px.png';
+
+          var avatarData = {
+            image: DEFAULT_PROFILE_IMAGE_PATH,
+            tooltipText: contributorName
+          };
+
+          var profileImageUrl = (
+            '/preferenceshandler/profile_picture_by_username/' +
+              contributorName);
+          // Returns a promise for the user profile picture, or
+          // the default image if user is not logged in or
+          // has not uploaded a profile picture, or
+          // the player is in preview mode.
+          $http.get(profileImageUrl).success(function(data) {
```

Comment:
@seanlip About the field 'contributor_names', yes it can be removed after changing it's only reference [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/player/LearnerViewBreadcrumb.js#L92) to `Object.keys(expInfo.human_readable_contributors_summary)`.